### ISSUE TITLE
Zulrah Helper Hotkeys Not Working

### ISSUE THREAD

**Issue by JamesYZFR1** [while-loop/runelite-plugins#42]
Basically title. I've set hotkeys for phase selection 1-3 and phase reset but they don't work. I have enter to chat enabled and I've tried a couple different hotkeys (shift + z/x/c, ctrl + z/x/c, q, F keys)

**Comment by while-loop** [comment]
cc @raiyni

**Comment by KBoumghar** [comment]
Same problem ; hotkeys simply don't work no matter which one I try. Chat is not opened by default (I have to press enter before being able to chat) and I tried disabling the "Key remapping" plugin, still doesn't work

**Comment by while-loop** [comment]
> Same problem ; hotkeys simply don't work no matter which one I try. Chat is not opened by default (I have to press enter before being able to chat) and I tried disabling the "Key remapping" plugin, still doesn't work

@KBoumghar are you in the Zulrah instance?
The hotkeys only work in the instance.

**Comment by KBoumghar** [comment]
> > Same problem ; hotkeys simply don't work no matter which one I try. Chat is not opened by default (I have to press enter before being able to chat) and I tried disabling the "Key remapping" plugin, still doesn't work
> 
> @KBoumghar are you in the Zulrah instance? The hotkeys only work in the instance.

@while-loop 
Yeah checking the code seems like it's what's happening, any reason to only enable them in `INSTANCE_ID` instead of `REGION_ID`, which is where the panel opens ... ? Really feel like this would make more sense, no (or only enable panel in `INSTANCE_ID`)? Feel like being able to test your keybinds works while not having to actively fight Zulrah at the same time makes much more sense
